scripts/simplify_network.py

```diff
@@ -243,7 +243,7 @@ def _aggregate_and_move_components(
     def replace_components(n, c, df, pnl):
         n.mremove(c, n.df(c).index)
 
-        import_components_from_dataframe(n, df, c)
+        import_components_from_dataframe(n, df.fillna(0), c)
```

Comment:
@martacki, it feels like this is mixing types explicitly, why was this raising an error before or a warning?